js/browzine.js

```diff
@@ -7,44 +7,62 @@
       $sceDelegateProvider.resourceUrlWhitelist(urlWhitelist);
   }]);
 
-// Add Browzine Links
-  app.controller('prmBriefResultAfterController', function($scope, $http) { 
+// Add Article In Context & Browzine Links
+  app.controller('prmSearchResultAvailabilityLineAfterController', function($scope, $http) { 
     var vm = this;
-    if (vm.parentCtrl.item.pnx.addata.doi)  {
-          vm.doi = vm.parentCtrl.item.pnx.addata.doi[0] || '';
-          var url = "https://yourserver.edu/primo/browzine/browzineArticleInContext?DOI=" + vm.doi;
-          $http.jsonp(url, {jsonpCallbackParam: 'callback'}).then(function(response) {
-          $scope.article = response.data;
-          }, function(error){
-                 console.log(error); //
+		if (vm.parentCtrl.result.pnx.addata.doi && vm.parentCtrl.result.pnx.display.type[0] == 'article')	{
+        	vm.doi = vm.parentCtrl.result.pnx.addata.doi[0] || '';
+        	var articleURL = "https://yourserver.edu/primo/browzine/browzineArticleInContext?DOI=" + vm.doi;
```

Comment:
Thanks Sarah!  Readme answers those questions now, excellent.

I will try setting something up with Amazon Lambda, because that sounds like a great use for it and would make setup much simpler than setting up a variety of EC2 boxes.